### PR TITLE
feat(nav): new functional nav components

### PR DESCRIPTION
**Pull request by alexsasharegan**

### CODE REVIEW

File: lib/components/form.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+    props: {
+        inline: {
+            type: Boolean,
+            default: false
+        }
+    },
+    render(h, { props, data, children }) {
+        return h("form", mergeData(data, { class: { "form-inline": props.inline } }), children);
+    }
+};
```
Comment (tmorehouse) [inline lib/components/form.js]: Thi will probably conflict with the updated form.vue component in the penndig form-validation branch

**Comment by tmorehouse** [comment]
👍

File: lib/components/navbar-brand.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    }
+});
+
+export default {
+    functional: true,
+    props: {
+        tag: {
+            type: String,
+            default: null
+        }
```
Comment (tmorehouse) [inline lib/components/navbar-brand.js]: Should this be the props as exported above?

Comment (alexsasharegan) [inline lib/components/navbar-brand.js]: Hmm... I set it to div! Let me see how that didn't copy over.

Comment (alexsasharegan) [inline lib/components/navbar-brand.js]: I see now. I moved props to an export and then didn't use them.

**Pull request merged**